src/commands/developers/init-distribution.js

```diff
@@ -0,0 +1,13 @@
+const initDistribution = require('../../scripts/init-distribution')
+
+module.exports = function(program) {
+  program
+    .command('init-distribution <name> <kernel>')
+    .description("Initialize your distribution project for standard libraries .\n  " +
+      "Provide a distribution <name> for your project\n  " +
+      "Provide the zeppelin_os <kernel> address where your releases are going to be published")
```

Comment:
Remember that backticks allow for [multiline strings](https://stackoverflow.com/a/805113/12791)